packages/atomic/src/components/atomic-result-placeholder-v1/atomic-result-placeholder.tsx

```diff
@@ -0,0 +1,97 @@
+import {Component, h, Prop} from '@stencil/core';
+import {
+  ResultDisplayLayout,
+  ResultDisplayDensity,
+  ResultDisplayImageSize,
+  getResultDisplayClasses,
+} from '../atomic-result-v1/atomic-result-display-options';
+
+const placeholderClasses = 'block bg-neutral w-full h-full rounded';
+
+/**
+ * The `atomic-result-placeholder` component provides an intermediate visual state that is rendered before the first results are available.
+ */
+@Component({
+  tag: 'atomic-result-placeholder-v1',
+  styleUrl: 'atomic-result-placeholder.pcss',
+  shadow: true,
+})
+export class AtomicResultPlaceholder {
```

Comment:
Could this have be an (internal) functional component instead ?

Reply:
I would like it to be a functional component, but unfortunately functional components cannot have a ShadowDOM, and I want the placeholder to be consistent with results.